examples/C/meson.build

```diff
@@ -5,4 +5,5 @@ subdir('ssmfe')
 spral_c_examples += [['lsmrs_c', files('lsmr.c')],
                      ['randoms_c', files('random.c')],
                      ['random_matrixs_c', files('random_matrix.c')],
-                     ['ssidss_c', files('ssids.c')]]
+                     ['ssidss_c', files('ssids.c')],
+                     ['ssids_topologys_c', files('ssids_topology.c')]]
```

Comment:
Is the extra "s" in the name intended?

Reply:
@amontoison why do we have all these extra s in the names for meson? Makes no sense to me.

Reply:
@jfowkes It is like GALAHAD and HSL, you have a prefix `s` for examples and `t` for tests.
You do that since 50 years at RAL!

Reply:
It's also what is generated by autotools.